deps.edn

```diff
@@ -133,7 +133,7 @@
   {:extra-deps  {com.github.seancorfield/depstar {:mvn/version "2.1.267"}}
    :extra-paths ["src/build"]} 
   :nvd
-  {:replace-deps {nvd-clojure/nvd-clojure {:mvn/version "1.9.0"}}
+  {:replace-deps {nvd-clojure/nvd-clojure {:mvn/version "2.0.0"}}
```

Comment:
I believe the scanner provides nvd itself, so no need to have this here (unless there is some other call in the makefile that needs it)

Reply:
It's for the `check-vuln` Makefile target. Neither this nor `check-vuln` are strictly necessary anymore, but as I said above I want to keep it for easy manual dev. Another way would be to get rid of this alias but keep `check-vuln`, having the latter use clojure.tools, but that would require the _user_ to ensure that they have nvm-clojure installed.